example/bundle-esm.js

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
Please change name of this file to the `bundle-esm.mjs`

Reply:
Went further, now `./example/` contains full visual of difference between CJS and ESM, written in code:
```
bundle-cjs.js
bundle-esm.js
bundle-cjs.cjs
bundle-esm.mjs
```